pwiz_tools/Skyline/SkylineGraphs.cs

```diff
@@ -907,6 +907,28 @@ private void charge4MenuItem_Click(object sender, EventArgs e)
         {
             _graphSpectrumSettings.ShowCharge4 = !_graphSpectrumSettings.ShowCharge4;
         }
+        private void synchMzScaleToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            if (_graphFullScan == null || !_graphFullScan.Visible || _graphSpectrum == null || !_graphSpectrum.Visible) return;

```

Comment:
This code needs work. 1. we don't use single line "if (condition) return;" These should be 2 lines.

We generally use "as" instead of "if (x is Type) t = (Type) x;

Assigning senter to source seems like it could be done using ?? followed by your other expression.

Reply:
Corrected.